apps/devhub/tests/test_views.py

```diff
@@ -1207,6 +1212,23 @@ def test_delete_and_recreate_credentials(self):
         assert new_key.key != old_key.key
         assert new_key.secret != old_key.secret
 
+    def test_retreive_credentials(self):
+        client = Client(enforce_csrf_checks=True)
+        assert client.login(username='regular@mozilla.com', password='password')
```

Comment:
I think `username=self.api_user.username` would read better to make it easier to follow where this user is set up